integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaRebuildTests.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.freeipa;
+
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.common.Status;
+import com.sequenceiq.it.cloudbreak.client.FreeIpaTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.freeipa.FreeIpaTestDto;
+import com.sequenceiq.it.cloudbreak.dto.telemetry.TelemetryTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+
+import static com.sequenceiq.it.cloudbreak.context.RunningParameter.key;
+import static com.sequenceiq.it.cloudbreak.context.RunningParameter.waitForFlow;
+
+public class FreeIpaRebuildTests extends AbstractE2ETest {
```

Comment:
could you add this to an e2e test suite? otherwise it won't run

Reply:
thanks, I missed that.

Done.